2022/chapters/procedure.adoc

```diff
@@ -1,157 +1,121 @@
+Points will also be given if the overall objective was not achieved.
+The duration of a trial is used as a tie-breaker (faster is better).
+
+=== Stage 1: Grab a Stationary Ball
 
-For each trial, a random robot starting location/orientation and a
-random ball starting position will be chosen. Both the robot and the
-ball will start stationary. There are no restrictions on the starting
-configuration other than that both robot and ball will be somewhere on
-the field.
+This stage is designed to be the easiest and to demonstrate basic sensing and control capabilities.
+The robot will need to detect a ball on the field, navigate to it, and place it on its dribbler.
 
-Before each run, the robot will begin in the halt state. A stop command
-will be issued to ready the robot, but the robot should not yet
-move. The run starts with a force start command, at which point the
-timer starts, and the robot may attempt to complete the task.
+==== Procedure
 
-The robot may move anywhere on the field within the walls. The ball
-may similarly move anywhere on the field throughout the time. Each
-trial will last a maximum of one minute.
+- The supervisor defines random starting positions for robot and ball, as well as a random orientation for the robot.
+- Robot and ball are moved to their starting positions
+- The trial is started
+- The trial is stopped after one minute or if the robot did not move for 10 seconds after moving at least once
 
 ==== Scoring
 
 - Touching the ball with any part of the robot -- +1
 - Touching the ball with the dribbler -- +1
 - Robot stopped with the ball touching the dribbler at the end of the run-- +1
 
-In the event of a tie between participants for this trial shorter
-times will be used to determine the ordering.
-
 === Stage 2: Scoring on an Empty Goal
 
-This stage is designed to build on stage 1, requiring that participant
-robots can also detect/localize on a goal without the SSL vision
-software. Like stage 1, the robot will need to detect a ball, move to
-it and manipulate it. The robot must put the ball into the goal to
-receive maximum points in the trial.
-
-The robot and ball will start the same as in Stage 1. This time, the
-goal is for the robot to score on an empty goal. Either goal on the
-field is acceptable. While scoring the robot may not enter the defense
-zone. Unlike regular gameplay, the ball does not need to stay on the
-ground in order for the goal to be counted.
-
-Before each run the robot will begin in the halt state. A stop command
-will be issued to ready the robot, but the robot should not yet
-move. The run begins with a force start command, at which point the
-timer starts and the robot may attempt to complete the task. Each
-trial will last a maximum of one minute. The trial will stop when the
-goal has occurred or time has elapsed.
+This stage requires robots to localize themselves as well as the ball on the playing field, move to the ball and score a goal.
```

Comment:
I don't think robots needs to localize themselves here, don't the ball and the goal are sufficient?

Reply:
true, changed it from themselves to goal